openedx/core/djangoapps/xblock/rest_api/urls.py

```diff
@@ -12,16 +13,16 @@
 app_name = 'openedx.core.djangoapps.xblock.rest_api'
 
 urlpatterns = [
-    url(r'^api/xblock/v2/', include([
-        url(r'^xblocks/(?P<usage_key_str>[^/]+)/', include([
+    path('api/xblock/v2/', include([
+        path('xblocks/<str:usage_key_str>/', include([
```

Comment:
This change shouldn't be necessary.
```suggestion
        path('xblocks/(?P<usage_key_str>[^/]+)/', include([
```